src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -10,6 +10,7 @@ import type { InjectIntlProvidedProps } from 'react-intl';
 import classNames from 'classnames';
 import getProp from 'lodash/get';
 import AsyncLoad from '../../common/async-load';
+import ContentAnswers from '../../../features/content-answers/ContentAnswers';
```

Comment:
I think this causes a [lint error](https://app.circleci.com/pipelines/github/box/box-ui-elements/16797/workflows/b723b842-cee2-4229-acb5-49cbd6ab3ac9/jobs/46927), I don't know if you can reference `features` from `elements`

Reply:
I think the lint error is related to importing a TS type in a vanilla JS file

Reply:
yeah I see one example which import files from elements to features
https://github.com/box/box-ui-elements/blob/217a39314c43f9b093c4ca55a45772a6f42a9416/src/elements/content-explorer/ContentExplorer.js#L25

Reply:
In this case, is it the only solution to change the `ContentAnswers` file in JS file?

Reply:
no you wouldn't need to convert to JS. you could create a flow file but it also seems acceptable in this repo to use `$FlowFixMe` comments for TS components
e.g. https://github.com/box/box-ui-elements/blob/4d71c6279313b24e9529c9f8b769348453a57226/src/elements/content-sidebar/DetailsSidebar.js#L16-L17